veg2hab/enums.py

```diff
@@ -546,3 +546,117 @@ def enkel_negatieven(self):
     @property
     def enkel_positieven(self):
         return LBKType._tuple_dict.value[self.name].enkel_positieven
+
+
+STR_MITSEN = [
```

Comment:
Dit is gewoon een kopie uit de mitsjson. Dat zou moeten kloppen toch?

Reply:
Kan zeker, maar dan moeten we die toevoegen aan de package_data. Zorgt er in de toekomst idd wel voor dat het beter te onderhouden is.